addons/auth_passkeys/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+auth_passkeys.access_auth_passkeys_key,access_auth_passkeys_key,model_auth_passkeys_key,base.group_user,1,1,1,1
```

Comment:
I would mimic what is done for `res.users.apikeys` regarding the access rights.
So, provide only read to users.
Maybe give the read access to portal users as well, if we want to give the possibility to portal to authenticate with pass keys. We allow them to create API keys so I don't see why we wouldn't give them the ability to sign in with passkeys.
https://github.com/odoo/odoo/blob/07f6d71e4dd3fac7b4cacc545819430aa48823cd/odoo/addons/base/security/ir.model.access.csv#L93-L94

Then, each time you need to add a new passkey, go through a creation wizard
https://github.com/odoo/odoo/blob/07f6d71e4dd3fac7b4cacc545819430aa48823cd/odoo/addons/base/models/res_users.py#L2166-L2178

And do not provide the create/write/delete access to user to manage they passkeys themselves.
Giving such right open attack vectors. Not only on their own passkeys but attackers on other user passkeys as well.